README.md

```diff
@@ -30,13 +30,42 @@ This blueprint assumes that you are familiar with GKE and
 To deploy this blueprint you need:
 
 - A [Google Cloud project](https://cloud.google.com/docs/overview#projects) with billing enabled.
-- An account with the [Project Owner role](https://cloud.google.com/iam/docs/understanding-roles#resource-manager-roles) on the project.
 - The `serviceusage.googleapis.com` must be enabled on the project. For more
   information about enabling APIs, see
   [Enabling and disabling services](https://cloud.google.com/service-usage/docs/enable-disable)
 - A Git repository to store the environment configuration.
 
-You create the infastructure using Terraform. The blueprint uses a local [Terraform backend](https://www.terraform.io/docs/language/settings/backends/configuration.html),
+### Service Account Roles & Permissions
+
+You can choose between **Project Owner** access (full access) or **Granular Access** for more fine-tuned permissions.
+
+#### Option 1: [Project Owner role](https://cloud.google.com/iam/docs/understanding-roles#resource-manager-roles)
+
+The service account will have full administrative access to the project.
+
+- `roles/owner`: Full administrative access to the project.
+
+---
+
+#### Option 2: Granular Access
+
+The service account will be assigned the following roles to limit access to required resources:
+
+- `roles/artifactregistry.admin`
+- `roles/browser`
+- `roles/cloudkms.admin`
+- `roles/compute.networkAdmin`
+- `roles/compute.securityAdmin`
+- `roles/compute.viewer`
+- `roles/container.clusterAdmin`
+- `roles/container.clusterViewer`
```

Comment:
I don't think we need this because we have `clusterAdmin`: https://cloud.google.com/iam/docs/understanding-roles#container.clusterAdmin

Reply:
Good point